packages/arb-token-bridge-ui/src/components/syncers/RetryableTxnsIncluder.tsx

```diff
@@ -31,16 +35,30 @@ export function RetryableTxnsIncluder(): JSX.Element {
       }
 
       const l1TxReceipt = new L1TransactionReceipt(depositTxReceipt)
-      const l1ToL2Msg = await l1TxReceipt.getL1ToL2Message(l2Provider)
-      const status = await l1ToL2Msg.status()
 
-      if (status !== L1ToL2MessageStatus.FUNDS_DEPOSITED_ON_L2) {
-        arbTokenBridge?.transactions?.fetchAndUpdateL1ToL2MsgStatus(
-          depositTx.txId,
-          l1ToL2Msg,
-          depositTx.asset === 'eth',
-          status
-        )
+      // TODO: Review this new change.
```

Comment:
@spsjvc @yahgwai linked to https://github.com/OffchainLabs/arbitrum-sdk/pull/206